src/77480.c

```diff
@@ -145,11 +145,15 @@ void func_800DFAAC(void) {
 
     check_input_midair_jump();
 
-    if (playerStatus->actionState != ACTION_STATE_SLIDING) {
+    if (playerStatus->actionState != ACTION_STATE_SLIDE) {
         collision_main_lateral();
         func_800E4508();
 
-        if ((collision_main_above() < 0) && (playerStatus->decorationList == 0) && (playerStatus->animFlags & 0x1000)) {
+        if (
```

Comment:
I don't really think this change is necessary. It'd be good to address automatic formatting soon so we don't have so much inconsistent formatting everywhere